src/main/resources/assets/api-docs.yaml

```diff
@@ -1965,6 +1965,31 @@ paths:
           description: No acknowledgement found for the given key.
         500:
           description: Error processing the request.
+    delete:
+      summary: ADMIN ONLY - Delete Acknowledgement by key
+      description: Deletes a specific acknowledgement for the current user by acknowledgement key.  Returns the deleted acknowledgement.
```

Comment:
any reason only for the current user? it seems plausible that the Admin API would allow deleting anyone's acknowledgement.

Reply:
Yes.  There may be compliance issues with doing so. Adding that to the scope of the ticket wasn't needed to verify the correct behavior in your work.  I didn't want/need to open that can of worms at this time.